solar-land/scss/layout/_layout.scss

```diff
@@ -0,0 +1,23 @@
+.container {
+  width: 100%;
+
+  @media (min-width: 640px) {
+    max-width: 640px;
```

Comment:
why px unit?! can we use em ?! otherwise it'll not work on Retina and high quality tools correctly

Reply:
it's just a temporary layout. we should to create a shared layout for containers, grid and some other general styles. I want to create it with team in refactoring phase.

Reply:
In this case, can you add TODO comment?
```
// TODO: temporary layout. we should to create a shared layout for containers, grid and some other general styles
```

Reply:
good idea.
I did it.